react/components/ProductImages/styles.css

```diff
@@ -24,18 +36,58 @@
   height: auto;
 }
 
-.carouselGaleryThumbs :global(.swiper-slide):not(:global(.swiper-slide-thumb-active)) .carouselThumbBorder {
+.carouselGaleryThumbs
+  :global(.swiper-slide):not(:global(.swiper-slide-thumb-active))
+  .carouselThumbBorder {
   border-width: 0px;
 }
 
 .carouselIconCaretRight,
 .carouselIconCaretLeft {
   stroke: white;
   stroke-width: 0.5px;
-  -webkit-filter: drop-shadow( 0px 1px 1px #CACBCC );
-  filter: drop-shadow( 0px 1px 1px #CACBCC );
+  -webkit-filter: drop-shadow(0px 1px 1px #cacbcc);
+  filter: drop-shadow(0px 1px 1px #cacbcc);
 }
 
-.carouselCursorDefault:hover{
+.carouselCursorDefault:hover {
   cursor: default;
-}
\ No newline at end of file
+}
+
+.slider {
+  position: absolute;
+  height: 5px;
+  overflow-x: hidden;
+}
+
+.subline {
+  position: absolute;
+  height: 5px;
+}
+.inc {
+  animation: increase 2s infinite;
```

Comment:
what is the purpose of these animations? animating `left` or `width` is generally not good for performance, it'd be better to stick with `transform` as much as possible

Reply:
I use this to animate the linear loader.